packages/components/src/presentation-components/instance-filter-builder/MultiTagSelect.tsx

```diff
@@ -133,7 +133,13 @@ function TagContainer<TOption, IsMulti extends boolean = boolean>({ children, ..
 function TagLabel<TOption, IsMulti extends boolean = boolean>({ children, ...props }: MultiValueGenericProps<TOption, IsMulti>) {
   return (
     <components.MultiValueLabel {...props} innerProps={{ ...props.innerProps, className: "iui-tag-label" }}>
-      {children === "" ? <i style={{ padding: 1 }}>Empty Value</i> : children}
+      {children === "" ? (
+        <span className="presentation-instance-filter-empty-property-text" style={{ padding: 1 }}>
```

Comment:
I think the new class could be combined with `iui-tag-label`. Also, padding could be assigned through SCSS:
```css
.iui-tag-label.presentation-instance-filter-empty-property-text {
  padding: 1;
}
```